layout/layout.h

```diff
@@ -741,11 +741,11 @@ M0_INTERNAL struct m0_layout *m0_layout_find(struct m0_layout_domain *dom,
  * If buffer is too small then returns the first very lid. If too large then
  * returns the layout id describing biggest possible unit_size.
  *
- * Always returns valid layout id.
+ * Returns -EINVAL if cannot find the right layout.
  */
-M0_INTERNAL uint64_t m0_layout_find_by_buffsize(struct m0_layout_domain *dom,
-						struct m0_fid *pver,
-						size_t buffsize);
+M0_INTERNAL int m0_layout_find_by_buffsize(struct m0_layout_domain *dom,
```

Comment:
I think it should be something line int64_t, in case we use high bits of layout identifier to encode something.

Reply:
Done.